nbs/models.timemixer.ipynb

```diff
@@ -62,7 +62,7 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #94.        DataEmbedding_wo_pos
</pre><p>You probably mean "Past Decomposable Mixing", and not "DataEmbedding_wo_pos"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1085/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Ahhh when ctrl+v and speed take over